rules/windows/process_creation/win_susp_explorer.yml

```diff
@@ -0,0 +1,32 @@
+title: Proxy Execution Via Explorer.exe
+id: 9eb271b9-24ae-4cd4-9465-19cfc1047f3e
+description: Attackers can use explorer.exe for evading defense mechanisms
+author: 'Furkan CALISKAN, @caliskanfurkan_, OSCD Community'
+status: experimental
+date: 10/02/2020
+references:
+    - https://twitter.com/bohops/status/1276356245541335048
+    - https://twitter.com/CyberRaiju/status/1273597319322058752
+tags:
+    - attack.defense_evasion
+    - attack.t1218
+logsource:
+    category: process_creation
+    product: windows
+detection:
+    selection1:
+        Image|endswith:
+            - \explorer.exe
+        CommandLine|contains:
+            - /root
+    selection2:
```

Comment:
Doesn't this trigger every time an 'explorer.exe' gets invoked from a 'cmd.exe'?

Reply:
Added as a known FP. Thank you.

Reply:
Hello @caliskanfurkan!

Thank you for your contribution! 

It seems that we did a mistake while compiling the table in issue #1014. 
[A rule that covers the selection1 of your rule](https://github.com/Neo23x0/sigma/blob/master/rules/windows/process_creation/win_susp_explorer_break_proctree.yml) was already developed by @Neo23x0 back in June 2019.
We missed that, I am sorry for it.
Could you please remove section1 from your rule?
If you are OK with that, please put "low" to the `level`, as this detection is not going to produce high fidelity alerts, but more like events for triage and further dynamic severity calculation. 

Thank you!

Reply:
Hello @caliskanfurkan! Could you please fix this?

Reply:
> Hello @caliskanfurkan! Could you please fix this?

Done